Tests/ProjectSpecTests/ProjectSpecTests.swift

```diff
@@ -97,7 +97,7 @@ class ProjectSpecTests: XCTestCase {
                 project.settings = invalidSettings
                 project.configFiles = ["invalidConfig": "invalidConfigFile"]
                 project.fileGroups = ["invalidFileGroup"]
-                project.localPackages = ["invalidLocalPackage" : LocalSwiftPackage(path: "invalidLocalPackage")]
+                project.packages = ["invalidLocalPackage" : SwiftPackage(kind: .local(path: "invalidLocalPackage"))]
```

Comment:
Once `SwiftPackage.Kind` is removed and `SwiftPackage` becomes the enum, this because simpler throughout:
```suggestion
                project.packages = ["invalidLocalPackage" : .local(path: "invalidLocalPackage")]
```